spec/factories/pockets.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :pocket do
-    number 'MyString'
-    state 1
+    serial_number { Faker::Number.number(10) }
+    state { %w[Unweighed Weighed Classified].sample }
```

Comment:
What do you think about to change this line by "state { Faker::Number.between(0, 2) }"?

Reply:
It is exactly the same, but i found my approach easier to understand for me. Is it better to you if i change it to that?

Reply:
It's not necessary, it's fine